### PR TITLE
Add serial and part number to the redfish alert

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

**Comment by tannoa2** [comment]
jenkins depends on : https://github.com/RackHD/RackHD/pull/817
jenkins depends on : https://github.com/RackHD/on-tasks/pull/465

**Comment by tannoa2** [comment]
@RackHD/veyron

File: lib/services/notification-api-service.js [hunk 1]
```diff
@@ -51,89 +51,89 @@ function NotificationApiServiceFactory(
...
-            progressData = _.pick(message, ['maximum', 'value', 'description']);
-        })
-        .then(function(){
-            return waterline.taskdependencies.findOne({taskId: message.taskId});
-        })
-        .then(function(task) {
-            if (_.isEmpty(_.get(task, 'graphId'))) {
-                throw new Errors.BadRequestError('Cannot find the task for taskId ' + message.taskId); //jshint ignore: line
-            }
+                message.value = parseInt(message.value);
```
Comment (geoff-reid) [inline lib/services/notification-api-service.js]: Please fix the indention issues.

File: lib/services/notification-api-service.js [hunk 2]
```diff
@@ -251,9 +254,12 @@ function NotificationApiServiceFactory(
         amqpMessage.data.ChassisName = null;
         amqpMessage.data.ServiceTag = null;
         amqpMessage.data.SN = null;
+        amqpMessage.data.SN = null;
```
Comment (BillyAbildgaard) [inline lib/services/notification-api-service.js]: lines 256 and 257 are the same

File: lib/services/notification-api-service.js (hunk 1 above)
Comment (BillyAbildgaard) [inline lib/services/notification-api-service.js]: +1

File: lib/services/notification-api-service.js [hunk 3]
```diff
@@ -274,6 +279,88 @@ function NotificationApiServiceFactory(
 
     }
 
+    function getSerialNumber(amqpMessage) {
+        //amqpMessage.data.originOfCondition = "Disk.Bay.1-0";
+        if (amqpMessage.data.OriginOfCondition != null) {
```
Comment (geoff-reid) [inline lib/services/notification-api-service.js]: You need to fix all hound issues

**Pull request merged**